test/example_data.dart

```diff
@@ -658,6 +675,21 @@ ReactionEvent reactionEvent(Reaction reaction, ReactionOp op, int messageId) {
   );
 }
 
+SubmessageEvent submessageEvent(
+  int messageId,
+  int senderId,
+  {required String content}
+) {
```

Comment:
nit: follow our standard formatting for signatures that have both positional and named parameters:
```suggestion
  int senderId, {
  required String content,
}) {
```

(here and a few other places)